### PR TITLE
[feature_1] feature_1 is implemented

### PR DESCRIPTION
**Pull request by t-tomaat**
Two new files were added:

- hello.txt
- info.txt

### CODE REVIEW

File: feature_1/hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello feature_1 !
```
Comment (ozanaksakal) [inline feature_1/hello.txt:1]: it is really good that you have decided to add this line of code

**Review by ozanaksakal — APPROVED** [review]
good job Mike

File: feature_1/hello.txt (hunk 1 above)
Comment (t-tomaat) [reply feature_1/hello.txt:1]: Thanks for your helpful comment.

File: feature_1/info.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+feature_1 is ready!
```
Comment (JehPark) [inline feature_1/info.txt:1]: Nice job, bro. You are my git hero.

**Review by JehPark — APPROVED** [review]
You are perfect lecturer for me.

File: feature_1/hello.txt (hunk 1 above)
Comment (Raj-kumar-007) [inline feature_1/hello.txt:1]: I have many new things about git & Git hub thanks :)

File: feature_1/info.txt (hunk 2 above)
Comment (Raj-kumar-007) [reply feature_1/info.txt:1]: Thank you :)

File: feature_1/hello.txt (hunk 1 above)
Comment (j-hey-fede) [reply feature_1/hello.txt:1]: I agree with @t-tomaat  - this line provided a lot of clarity.

Comment (RajeshDoolla) [reply feature_1/hello.txt:1]: It is really good that you decided to add this one

**Review by mikegithubber — APPROVED** [review]
Merging. Thanks for contribution

**Pull request merged**